packages/eden/src/plugins/index.ts

```diff
@@ -1,4 +1,4 @@
-import type Elysia from 'elysia'
+import type ElysiaCore from 'elysia'
```

Comment:
Not sure what the benefit of not calling it `Elysia`. I think it would be best if it was just included with the import on the next line.

```ts
import type { AnyElysia, Elysia } from 'elysia'
```